### PR TITLE
CB-55: Add OptGroupSelectField for frequently used languages

### PR DESCRIPTION
**Pull request by man-of-code**
Added `OptGroupSelectField` from `wtforms-components` for separating frequently used languages.

### CODE REVIEW

**Comment by ferbncode** [comment]
@brainzbot add to whitelist

File: critiquebrainz/db/review.py [hunk 1]
```diff
@@ -40,6 +40,22 @@ def to_dict(review, confidential=False):
     return review
 
 
+def get_top_languages():
+    """
+        List of the top ten languages (only the language codes) in CritiqueBrainz.
+    """
```
Comment (ferbncode) [inline critiquebrainz/db/review.py]: Make sure that the docstring matches the guidelines: https://github.com/metabrainz/guidelines/blob/master/Python.md#docstrings

File: critiquebrainz/frontend/forms/review.py [hunk 2]
```diff
@@ -20,13 +21,32 @@ def __call__(self, form, field):
...
 
 
-# Loading supported languages
-languages = []
-for language_code in supported_languages:
-    try:
-        languages.append((language_code, Locale(language_code).language_name))
-    except UnknownLocaleError:
-        languages.append((language_code, pycountry.languages.get(iso639_1_code=language_code).name))
+def get_languages():
```
Comment (ferbncode) [inline critiquebrainz/frontend/forms/review.py]: It would be great if you could add a docstring for this function too.

**Comment by ferbncode** [comment]
The tests are failing because the `create_app()` function is called in `python manage.py init_db --skip-create-db --test-db` command in test container which fails to recognize `db` (which the `get_top_language()` function is using). We need to figure out a way to fix this in tests.

We can also change the way `frequently_used_languages` is populated. We can maintain a script for updating frequently used languages (like in MB: https://github.com/metabrainz/musicbrainz-server/blob/master/admin/SetLanguageFrequencies)

**Comment by paramsingh** [comment]
@ferbncode: Seems like MB has a cron job running daily to calculate language frequencies: https://github.com/metabrainz/musicbrainz-server/blob/master/admin/cron/daily.sh#L87, we could do something like that too.

**Comment by man-of-code** [comment]
Okay, I'll add the `doctsrings` according to the guidelines. Thanks for reviewing @ferbncode .

**Comment by ferbncode** [comment]
I think the method for calculating `frequently_used_languages` should be similar to MusicBrainz. @man-of-code , it would be awesome if you could write a script that executes the exact same query, updates cache (store the result in a namespace 'frequent-languages'). Then we can add that script to cron, so that it gets updated every day (or week).

**Comment by man-of-code** [comment]
@ferbncode I'll get to it, right away.

**Comment by paramsingh** [comment]
Closing this for now, we can do a different PR for calculating frequently used languages first.

**Pull request closed without merging**